DxDispatch/src/dxdispatch/DmlDispatchable.cpp

```diff
@@ -140,12 +420,12 @@ void DmlDispatchable::Initialize()
     THROW_IF_FAILED(m_device->DML()->CreateBindingTable(&bindingTableDesc, IID_PPV_ARGS(&bindingTable)));
 
     // Inputs flagged OWNED_BY_DML must be bound during initialization (and only initialization).
-    if (!m_initBindings.empty())
+    if (m_isSerializedGraph && !m_serializedGraphBindings.empty())
```

Comment:
Same comment as in the `Bind` method. If we keep only variable for `initBinding` then we don't need separate code path for serializedGraphDispatchable and dmlDispatchable.